vllm/entrypoints/openai/protocol.py

```diff
@@ -42,23 +42,31 @@ class OpenAIBaseModel(BaseModel):
     # OpenAI API does allow extra fields
     model_config = ConfigDict(extra="allow")
 
+    # Cache class field names
+    field_names: ClassVar[Optional[Set[str]]] = None
```

Comment:
There was noticeable overhead creating this set every time one of these objects is instantiated.